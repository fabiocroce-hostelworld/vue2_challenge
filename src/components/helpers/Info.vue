<template>
  <v-alert dense type="info">
    {{ message }}
  </v-alert>
</template>

<script lang="ts">
import { defineComponent } from "@vue/composition-api"

export default defineComponent<{ message: string }>({
  name: "Info",
  props: {
    message: {
      type: String,
      required: true
    }
  }
})
</script>
