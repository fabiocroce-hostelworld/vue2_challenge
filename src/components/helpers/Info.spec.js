import { shallowMount, createLocalVue } from "@vue/test-utils"
import Info from "@/components/helpers/Info.vue"

describe("Info", () => {
  it("Snapshot test", () => {
    const wrapper = shallowMount(Info, {
      localVue: createLocalVue(),
      propsData: {
        message: "test"
      }
    })
    expect(wrapper.findAll(".info").length).toBe(0)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
