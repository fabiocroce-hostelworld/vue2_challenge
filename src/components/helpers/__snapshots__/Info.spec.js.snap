// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Snapshot test 1`] = `
<v-alert-stub tag="div" value="true" closelabel="$vuetify.close" dense="true" closeicon="$cancel" icon="" type="info">
  test
</v-alert-stub>
`;
